templates/base.html

```diff
@@ -28,6 +32,51 @@
     <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/stylesheet/style.min.css">
   {% endif %}
 
+  <!-- DARK THEME STYLES -->
```

Comment:
Remove these comments, they do not help much, you could used Jinja comments if you wish to keep it